media-libs/libplacebo/libplacebo-4.192.1-r2.ebuild

```diff
@@ -26,11 +26,11 @@ IUSE="glslang lcms +opengl +shaderc test unwind +vulkan"
 REQUIRED_USE="vulkan? ( || ( glslang shaderc ) )"
 RESTRICT="!test? ( test )"
 
-RDEPEND="glslang? ( dev-util/glslang[${MULTILIB_USEDEP}] )
+RDEPEND="glslang? ( <dev-util/glslang-1.3.231:=[${MULTILIB_USEDEP}] )
 	lcms? ( media-libs/lcms:2[${MULTILIB_USEDEP}] )
 	opengl? ( media-libs/libepoxy[${MULTILIB_USEDEP}] )
 	shaderc? ( >=media-libs/shaderc-2017.2[${MULTILIB_USEDEP}] )
-	unwind? ( sys-libs/libunwind:= )
+	unwind? ( sys-libs/libunwind:=[${MULTILIB_USEDEP}] )
```

Comment:
While here, we may want to do the llvm-libunwind dance?

Reply:
I meant to but didn't feel like bothering adding new checks and upstreaming it, it uses pkg-config

Reply:
On that note, why does llvm-libunwind not have a `.pc` file :thinking:

It's really the only issue here.

Reply:
👀